README.md

```diff
@@ -345,6 +345,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [port_midi](https://github.com/jimm/crystal_port_midi) - Crystal C bindings for the PortMIDI cross-platform MIDI I/O library
  * [retriable.cr](https://github.com/Sija/retriable.cr) - Simple DSL to retry failed code blocks
  * [simple_retry](https://github.com/spider-gazelle/simple_retry) - Simple tool for retrying failed code blocks
+ * [sslscan.cr](https://github.com/NeuraLegion/sslscan.cr) - Crystal shard wrapping the rbsec/sslscan utility
```

Comment:
Would the `Networking` section be a better fit?

Reply:
Well, perhaps it would, but the essence of the shard is about the enumerating ssl/tls weaknesses for given host, so the main topic here is `Security`, but more on the offence side than the one found atm in the README.